beam_sink/sinks/mysql.py

```diff
@@ -73,6 +90,9 @@ def finish_bundle(self) -> None:
 
 
 class _PutFn(beam.DoFn):
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'e')](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)